src/components/dimensions/DimensionFilterRow.js

```diff
@@ -46,7 +46,7 @@ class DimensionFilterRow extends Component {
             <div className={classes.container}>
                 <DimensionSelect
                     dimension={dimension}
-                    onChange={dimension => this.onChange(dimension.id, items)}
+                    onChange={dimension => this.onChange(dimension.id)}
```

Comment:
Can we change the name of this variable to `dimensionSelectEntry` or something?  It's quite confusing looking at this function, since `dimension` in the outer scope is the dimension `id` but `dimension` in the inner scope is an object, and both are passed at different times to `onChange`

Reply:
Agreed. Changed "selectedDimension".